api/v1alpha1/replicationdestination_types.go

```diff
@@ -192,6 +192,12 @@ type ReplicationDestinationResticSpec struct {
 	// accessModes can be used to set the accessModes of restic metadata cache volume
 	//+optional
 	CacheAccessModes []v1.PersistentVolumeAccessMode `json:"cacheAccessModes,omitempty"`
+	// Previous specifies the number of image to skip before selecting one to restore from
+	//+optional
+	Previous int32 `json:"previous,omitempty"`
+	// RestoreAsOf refers to the backup that is most recent as of that time.
+	//+optional
+	RestoreAsOf *string `json:"restoreAsOf,omitempty"`
```

Comment:
You'll want to use the [kubebuilder markers](https://book.kubebuilder.io/reference/markers/crd-validation.html) to get the `format: date-time` to be added to the CRD.